### PR TITLE
Start scratch of be_kind.md

### PR DESCRIPTION
**Pull request by marcospb19**

### CODE REVIEW

**Comment by marcospb19** [comment]
With very opinionated points on how to behave in the Rust open-source development community :slightly_smiling_face: .

I think it's lacking talking about being kind to beginners, as they have a hard time starting to contribute, usually because of insecurity. 

EDIT: I also forgot to mention the Code Of Conduct.

**Comment by nikomatsakis** [comment]
Oh, interesting! I definitely prefer this version. I've been thinking about how to alter "Be kind" based on @matklad's feedback. Some of the themes bounding around in my head were:

* Benefit of the doubt -- people are rarely out to get you, so the question is, can we find what they *are* after? There's probably something useful or important there. 
* Being forgiving -- people make mistakes

These may just be separate principles.

Maybe we should quote Maya Angelou: “I've learned that people will forget what you said, people will forget what you did, but people will never forget how you made them feel.”

File: src/how_to_rustacean/be_kind.md
Comment (nikomatsakis) [inline src/how_to_rustacean/be_kind.md]:
```suggestion
Rust is being built and used by different people from around the globe and everyone has their own opinion on what is the best for the language, so disagreements are inevitable.
```

**Review by nikomatsakis — CHANGES_REQUESTED** [review]
Minor nit

File: src/how_to_rustacean/be_kind.md
Comment (marcospb19) [reply src/how_to_rustacean/be_kind.md]: Updated :+1:.

**Comment by marcospb19** [comment]
> Some of the themes bounding around in my head were:
>
> - Benefit of the doubt -- people are rarely out to get you, so the question is, can we find what they _are_ after? There's probably something useful or important there.
> - Being forgiving -- people make mistakes
>
> These may just be separate principles.

The first one seems more like a tip to dealing better with people than a principle.

I think that both are close enough to go on the same page.

The _"being forgiving"_ theme fits better with "**Be kind and considerate**" principle than _"benefit of the doubt"_, however, this page is already talking about how to deal with other people, so it's probably the best place to insert more thoughts about it.

**Comment by marcospb19** [comment]
I left this PR on

**Comment by marcospb19** [comment]
I liked the quote, but don't know exactly how to insert it in the text, I left this PR with "Allow edits from maintainers" checked, so feel free to make any changes before merging.

**Pull request closed without merging**
